asm/rom.s

```diff
@@ -1,5 +1,8 @@
 /* rom.s TODO: Disassemble */
+.include "asm/macros.inc"
 
 .section .text
 
+	thumb_func_start NdsMain
+NdsMain:
```

Comment:
Should be renamed to `NitroMain` to match decomp.

Reply:
We can't see your Ghidra definitions

Reply:
This name is from NitroSDK